library/include/ck/library/reference_tensor_operation/cpu/reference_batched_gemm_transpose.hpp

```diff
@@ -0,0 +1,141 @@
+#ifndef REFERENCE_BATCHED_GEMM_TRANSPOSE_HPP
```

Comment:
I think we don't need a reference operator for every DeviceOp.

Here is an example

https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/56adf7e9cc4fcf6592151281a727e96b625bc54f/example/04_gemm_add_add_fastgelu/gemm_add_add_fastgelu_xdl_fp16.cpp#L212-L237

Reply:
Let us leave it to unifying Gemm kernels.